apps/web-giddh/src/app/shared/sales-person/sales-person.component.ts

```
@@ -0,0 +1,234 @@
+
+    /**
+     * Handle sales person action
+     *
+     * @param {SalesPersonActionEnum} action
+     * @param {any} [element]
+     * @memberof SalesPersonComponent
+     */
+    public salesPersonAction(action: SalesPersonActionEnum, element?: any): void {
+        switch (action) {
+            case SalesPersonActionEnum.CREATE:
+                const model = this.salesPersonForm?.value;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)